packages/core/echo/live-object/src/ref/ref-array.ts

```diff
@@ -0,0 +1,32 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import { type BaseObject, type ObjectId, Ref } from '@dxos/echo-schema';
```

Comment:
wait, live object depends on echo schema? why is that?

Reply:
conceptually that seems better to me: you first define the schema of the data, and then you have the API to manipulate that data. But im not set on this